install/kubernetes/helm/istio/values.yaml

```diff
@@ -144,6 +144,7 @@ global:
         requests:
           cpu: 10m
           memory: 10Mi
+    # use fully qualified iamge names for alternate path to proxy.
```

Comment:
unrelated change?

Reply:
yeah.. just updated the comment.. found this missing while testing this..

Reply:
Ok there is a typo in it. s/iamges/images

Reply:
Thanks.. Fixed